plugin/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go

```diff
@@ -82,17 +82,123 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 		  "kind": "Policy",
 		  "apiVersion": "v1",
 		  "predicates": [
-		    {"name": "PodFitsHostPorts"}
+		    {"name": "MatchNodeSelector"},
+		    {"name": "PodFitsHostPorts"},
+		    {"name": "PodFitsResources"},
+		    {"name": "NoDiskConflict"},
+		    {"name": "HostName"}
```

Comment:
just saw the comment. we want to make sure all supported options still work, not just the defaults